src/components/AnimatedSignal/commonUtils.ts

```diff
@@ -0,0 +1,26 @@
+export const makeArcPath = (
+  x: number,
```

Comment:
@het-a-simform Rename this to ArcPathUtils.ts

Reply:
Change method name to getArcPath instead of makeArcPath

Reply:
Resolved